src/main/java/com/conveyal/r5/analyst/scenario/RemoveStops.java

```diff
@@ -47,6 +48,16 @@
     /** Stops to remove from the routes. */
     public Set<String> stops;
 
+    /**
+     * Number of seconds (in addition to dwell time, if any) to remove from the schedule at each removed stop,
+     * to account for acceleration, deceleration, merging, and so on.
```

Comment:
Maybe we should say "vehicle acceleration and deceleration" and "merging into traffic" to make it clearer that we're talking about time saved by physical changes in bus operations and not something in code.